src/Host/Client/Impl/rtvs/R/util.R

```diff
@@ -53,6 +53,18 @@ toJSON <- function(obj) {
   call_embedded("toJSON", obj)
 }
 
+create_blob <- function(obj) {
+    call_embedded("create_blob", obj)
+}
+
+get_blob <- function(blob.id) {
```

Comment:
Will update in the next iteration.

---
In reply to: [70860104](https://github.com/Microsoft/RTVS/pull/2076#discussion_r70860104) [](ancestors = 70860104)